src/App.tsx

```diff
@@ -1,26 +1,337 @@
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-loading-todos#react-todo-app-load-todos">
-          React Todo App - Load Todos
-        </a>
-      </p>
-
-      <p className="subtitle">Styles are already copied</p>
-    </section>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <header className="todoapp__header">
+          {/* this button should have `active` class only if all todos are completed */}
+          <button
+            type="button"
+            className="todoapp__toggle-all active"
+            data-cy="ToggleAllButton"
+          />
+
+          {/* Add a todo on form submit */}
+          <form onSubmit={handlerAddTodo}>
+            <input
+              ref={inputRef}
+              data-cy="NewTodoField"
+              value={todoTitle}
+              onChange={handleTitleChange}
+              type="text"
+              className="todoapp__new-todo"
+              placeholder="What needs to be done?"
+              disabled={isLoadingTodo !== null}
+            />
+          </form>
+        </header>
+
+        {todos.length > 0 && (
+          <>
+            <TodosList
+              isLoadingTodo={isLoadingTodo}
+              isLoading={isLoading}
+              tempoTodo={tempoTodo}
+              todos={filteredTodos}
+              handleChangeStatus={handleChangeStatus}
+              handleDeleteTodo={handleDeleteTodo}
+            />
+
+            <footer className="todoapp__footer" data-cy="Footer">
+              <span className="todo-count" data-cy="TodosCounter">
+                {leftItems} items left
+              </span>
+
+              <nav className="filter" data-cy="Filter">
+                <a
+                  href="#/"
+                  className={classNames('filter__link', {
+                    selected: filterTerm === '',
+                  })}
+                  data-cy="FilterLinkAll"
+                  onClick={() => handleFiltered('')}
+                >
+                  All
+                </a>
+
+                <a
+                  href="#/active"
+                  className={classNames('filter__link', {
+                    selected: filterTerm === 'active',
+                  })}
+                  data-cy="FilterLinkActive"
+                  onClick={() => handleFiltered('active')}
+                >
+                  Active
+                </a>
+
+                <a
+                  href="#/completed"
+                  className={classNames('filter__link', {
+                    selected: filterTerm === 'completed',
+                  })}
+                  data-cy="FilterLinkCompleted"
+                  onClick={() => handleFiltered('completed')}
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method. Also move this logic to the separate component